dali/operators/reader/loader/webdataset/tar_utils.cc

```diff
@@ -202,6 +215,23 @@ inline bool TarArchive::ParseHeader() {
   } else {
     filename_ = th_get_pathname(ToTarHandle(handle_));
     filesize_ = th_get_size(ToTarHandle(handle_));
+
+    if (TH_ISREG(ToTarHandle(handle_))) {
```

Comment:
```
auto tar_handle = ToTarHandle(handle_)
```
and reuse later?

Reply:
but that function is just a reinterpret cast and storing it in a variable would be pointless